cvat-ui/src/reducers/review-reducer.ts

```diff
@@ -131,8 +131,28 @@ export default function (state: ReviewState = defaultState, action: any): Review
                                 }
                             });
                         });
-                        mainConflict.description = descriptionList.join(', ');
-                        mergedFrameConflicts.push(mainConflict);
+
+                        // decorate the original conflict to avoid changing it
+                        const description = descriptionList.join(', ');
+                        const visibleConflict = new Proxy(mainConflict, {
+                            get(target, prop, receiver) {
+                                if (prop === 'description') {
+                                    return description;
+                                }
+
+                                // By default, it looks like Reflect.get(target, prop, receiver)
+                                // which has a different value of `this`
+                                const value = target[prop];
+                                if (value instanceof Function) {
+                                    return function (...args) {
+                                        return value.apply(this === receiver ? target : this, args);
+                                    };
+                                }
+                                return value;
```

Comment:
Not sure what is problem with using Reflect here.
In this example context is preserved:
```js
(() => {
    const obj = { a: 5, test: function () { return this.a } };
    const proxy = new Proxy(obj, {
        get(target, prop, receiver) {
            if (prop === 'def') { return 'default' }
            return Reflect.get(target, prop, receiver);
        }
    })
    console.log(proxy.def) // default
    console.log(proxy.test()) // 5
})()
```